### PR TITLE
Feature nxp 27523 docker server

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +65,14 @@ npm run build
 
 Build and optimize the current project, ready for deployment. This includes linting as well as vulcanization, image, script, stylesheet and HTML optimization and minification.
 
+#### Run with Docker Compose
+
+```sh
+docker-compose run --build
```
Comment (Gabez0r) [inline README.md]:
```suggestion
docker-compose up --build
```

Comment (Gabez0r) [reply README.md]: Perhaps state that we need to run `npm run build` first?

File: README.md [hunk 2]
```diff
@@ -47,6 +65,14 @@ npm run build
...
+```
+
+This builds the `nuxeo-web-ui` Docker image and starts the Docker compose cluster.
```
Comment (Gabez0r) [inline README.md]: We should probably add the URL people should be hitting after it's running,

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,3 @@
+NUXEO_WEB_UI_VERSION=3.0.0-SNAPSHOT
+NUXEO_VERSION=10.10
```
[the remark this answers is not in the document]
Comment (nmpcunha) [reply .env.sample]: You should... did you tried `master`?

File: Jenkinsfile [hunk 4]
```diff
@@ -30,10 +30,18 @@ void setGitHubBuildStatus(String context, String message, String state) {
   ])
 }
 
+def WEBUI_VERSION
```
[the remark this answers is not in the document]
Comment (nelsonsilva) [reply Jenkinsfile]: `.env.sample` is just an example file, no telling what values ppl will use for `NUXEO_WEB_UI_VERSION`, could be custom version just to play around with docker-compose locally.§ 
This variable in the other hand is the version in the `package.json`.

File: .env.sample (hunk 3 above)
[the remark this answers is not in the document]
Comment (Gabez0r) [reply .env.sample]: I did not. This was rather a question on why we are using 10.10 on a Web UI version targeting 11.1-SNAPSHOT.

[the remark this answers is not in the document]
Comment (nelsonsilva) [reply .env.sample]: 'cause it works and this is just a sample `.env` but I'll check for a newer image (not sure we have it in the official docker registry)

[the remark this answers is not in the document]
Comment (Gabez0r) [reply .env.sample]: fair enough

File: README.md (hunk 2 above)
Comment (nelsonsilva) [reply README.md]: 👍

File: .env.sample (hunk 3 above)
[the remark this answers is not in the document]
Comment (nelsonsilva) [reply .env.sample]: changed to `master` as suggested 👍

**Pull request merged**